ramls/loan.json

```diff
@@ -253,6 +253,10 @@
         }
       }
     },
+    "patronGroupAtCheckout": {
+      "description": "Patron Group name at checkout",
+      "type": "string"
```

Comment:
@marc-johnson Could you please give more information on justifications to return an object against a field? In this case it was desided to return just a name field because the ID was not planned to be used in the presentation.

Reply:
@mrdpopov sure, I can try.

Using a string limits the extensibility of the API. If, in the future, more properties of the patron group are needed, then a breaking change would be needed to change the string to an object.

If the name is definitely the only property needed, then the name of the property should explicitly reflect that, e.g. patronGroupNameAtCheckout.

I prefer the object approach as it provides extensibility without committing to future details, and more explicit semantic description, at the slight tradeoff of access.

Given that the decision in the previous work was to store the ID, that suggests to me that there is a desire for more information about the patron group, otherwise only the name could’ve been stored.

Reply:
I changed it to the object. Example:
` "patronGroupAtCheckout" : {
      "id" : "ad0bc554-d5bc-463c-85d1-5562127ae91b",
      "name" : "graduate"
    }`

Reply:
The schema is still showing a string for me, have the changes not been pushed yet?

Reply:
@marcjohnson-kint Yes, you are right. I changed everywhere but forget this JSON-file. 

I pushed it with changes regarding @mrdpopov comments.

Reply:
Can you add           "readonly" : true to properties and       "additionalProperties": false to the object to follow Folio convention?

Reply:
Yes, sure.